internal/command/cliconfig/config_unix.go

```diff
@@ -22,6 +22,11 @@ func configFile() (string, error) {
 	newConfigFile := filepath.Join(dir, ".tofurc")
 	legacyConfigFile := filepath.Join(dir, ".terraformrc")
 
+	if xdgDir := os.Getenv("XDG_CONFIG_HOME"); xdgDir != "" && !pathExists(legacyConfigFile) && !pathExists(newConfigFile) {
+		// a fresh install should not use terraform naming
+		return filepath.Join(xdgDir, "opentofu", "tofurc"), nil
```

Comment:
Is the omission of a dot in `.tofurc` intentional here?

Reply:
Yes. From my understanding the dot is not needed for files in XDG because they (usually) are already in a dot directory, e.g. `~/.config`. For example, git will use `XDG_CONFIG_HOME/git/config`.

Reply:
Makes sense, thanks!